### PR TITLE
Move test directories out of the src directory

### PR DESCRIPTION
**Pull request by andyhite**
This should fix the build issues reported in https://github.com/ShipEngine/firebase-extensions/issues/19

### CODE REVIEW

**Review by forksofpower — APPROVED** [review]
lgtm!

File: extensions/purchase-label/functions/package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
     "mocha": "^9.1.1",
     "shipengine": "^1.0.0",
     "shipengine-firebase-common-lib": "^1.0.0",
+    "ts-mocha": "^8.0.0",
```
Comment (Salakar) [inline extensions/purchase-label/functions/package.json:28]: Should this, chai & mocha be moved to `devDependencies`? Not a major issue just will get installed when deploying the extension - but not get used 

Same on other extensions below

**Review by Salakar — APPROVED** [review]
Minor comment re `dependencies` vs `devDependencies` but other than that LGTM

File: extensions/purchase-label/functions/package.json (hunk 1 above)
Comment (andyhite) [reply extensions/purchase-label/functions/package.json:28]: Ah dang, good catch. I used `yarn add` and forgot to add `--dev`. I'll update that.

**Comment by andyhite** [comment]
Updated! merging now

**Pull request merged**